m/configDTOutputNetCDFL2.m

```diff
@@ -620,7 +648,131 @@
     'grid_coordinates' []
     'grid_method'      [] };
 
-
+  %% Variables specific to SeaExplorer
+  var_attr_list.mfl_v1 = {
+      'long_name'     'MiniFluo raw path 1'
+      'standard_name' 'tryptophan_or_pyrene_measurement'
+      'units'         'counts'
+      'coordinates'   'time depth latitude longitude'
+      '_FillValue'    default_fill_value
+      'sources'       []
+      'conversion'       []
+      'grid_sources'     []
+      'grid_coordinates' []
+      'grid_method'      [] };
+  
+  var_attr_list.mfl_v2 = {
+      'long_name'     'MiniFluo raw path 2'
+      'standard_name' 'phenantrene_or_fluoren_measurement'
```

Comment:
Ok, and please remember that this is just the default **example** configuration,
the final user can make any modifications to adjust it to his needs,
so it is not that relevant nor engraved in stone. I was just afraid it was a typo,
and wanted to avoid a spurious later commit to fix it.